src/ucp/proto/proto_multi.h

```diff
@@ -111,6 +111,11 @@ typedef struct {
     /* Maximal number of lanes to select */
     ucp_lane_index_t               max_lanes;
 
+    /* Minimal chunk size. This field is meaningful only for multi-fragment
+     * protocols. It defines the minimal size of the fragment to split into
+     * several parts. The goal is to not split below this limit */
```

Comment:
```suggestion
    /* Minimal chunk size. It defines the minimal size of the fragment to split into
     * several parts. The goal is to not split below this limit */
```
I'd remove this phrase, as it is already defined only for multi protocols